aspnetcore/blazor/components/virtualization.md

```diff
@@ -106,7 +106,24 @@ private async ValueTask<ItemsProviderResult<Employee>> LoadEmployees(
 }
 ```
 
-<xref:Microsoft.AspNetCore.Components.Web.Virtualization.Virtualize%601.RefreshDataAsync%2A?displayProperty=nameWithType> instructs the component to rerequest data from its <xref:Microsoft.AspNetCore.Components.Web.Virtualization.Virtualize%601.ItemsProvider%2A>. This is useful when external data changes. There's no need to call <xref:Microsoft.AspNetCore.Components.Web.Virtualization.Virtualize%601.RefreshDataAsync%2A> when using <xref:Microsoft.AspNetCore.Components.Web.Virtualization.Virtualize%601.Items%2A>.
+<xref:Microsoft.AspNetCore.Components.Web.Virtualization.Virtualize%601.RefreshDataAsync%2A?displayProperty=nameWithType> instructs the component to rerequest data from its <xref:Microsoft.AspNetCore.Components.Web.Virtualization.Virtualize%601.ItemsProvider%2A>. This is useful when external data changes. There's no need to call <xref:Microsoft.AspNetCore.Components.Web.Virtualization.Virtualize%601.RefreshDataAsync%2A> when using <xref:Microsoft.AspNetCore.Components.Web.Virtualization.Virtualize%601.Items%2A>. 
+
+Note that `RefreshDataAsync` only updates the virtualize component's data content, without causing a re-render. In the event `RefreshDataAsync` is invoked from a Blazor event handler or component lifecycle method, this is not an issue since a render is automatically triggered at the event of the method. However, if `RefreshDataAsync` is triggered separately from a background task or event, a call to `StateHasChanged` should also be made to update the UI.
```

Comment:
The phrase "at the event of the method" is strange wording. Is that what you meant? Did you mean "at the end of the method"?

Reply:
... and which method does "this" refer to because it could be either `RefreshDataAsync` or the event handler/lifecycle method.

Reply:
Oops, I meant at the end of the event or lifecycle method.

Reply:
Thanks ... I'll make a few tiny edits and merge this tomorrow morning.